From 6bdacf6aa5cf4d60b86a11629f799b7143c9abe4 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Wed, 25 May 2016 03:25:10 -0400
Subject: [PATCH 28/62] default repository setting does not support mirrorlist

It is yum based, and hard to implement in smart.
The requirement is not clear also.

Upstream-Status: Inappropriate [wrlinux specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 pyanaconda/ui/gui/spokes/source.py | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/pyanaconda/ui/gui/spokes/source.py b/pyanaconda/ui/gui/spokes/source.py
index 7704553..5dfedb0 100644
--- a/pyanaconda/ui/gui/spokes/source.py
+++ b/pyanaconda/ui/gui/spokes/source.py
@@ -1157,7 +1157,9 @@ class SourceSpoke(NormalSpoke, GUISpokeInputCheckHandler):
         # the currently selected protocol.
         self._proxyButton.set_sensitive(self._http_active() or self._mirror_active())
         self._nfsOptsBox.set_visible(self._nfs_active())
-        self._mirrorlistCheckbox.set_visible(self._http_active())
+
+        # Default repository setting does not support mirrorlist
+        self._mirrorlistCheckbox.set_visible(False)
         self._setup_no_updates()
 
         # Any changes to the protocol combo box also need to update the check to see
-- 
2.7.4

